fmriprep/workflows/base.py

```diff
@@ -275,7 +275,7 @@ def init_single_subject_wf(subject_id):
 
         workflow.connect([
             (anat_preproc_wf, func_preproc_wf,
-             [(('outputnode.t1w_preproc', _pop), 'inputnode.t1w_preproc'),
+             [('outputnode.t1w_preproc', 'inputnode.t1w_preproc'),
```

Comment:
You only modified the N4-only workflow -- are you positive this will work with the default brain extraction workflow?

Reply:
Okay, I just saw #498.

Reply:
local testing with https://github.com/poldracklab/smriprep/pull/174 seems to work with both branches of `--skull-strip-t1w` - `brain_extraction_wf` and `n4_only_wf`